docs/automation/README.md

```diff
@@ -23,15 +23,37 @@ We expect your application will be managing endpoints to generate and accept a n
 
 An example of this is below:
 
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
+<template v-slot:py>
+
+``` py
+...
```

Comment:
I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

Reply:
will add in next release